new-cdc/eventuate-local-java-cdc-connector-unified/src/main/java/io/eventuate/local/unified/cdc/pipeline/CdcPipelineConfigurator.java

```diff
@@ -61,18 +56,18 @@
   @Autowired
   private DefaultSourceTableNameResolver defaultSourceTableNameResolver;
 
+  @Autowired
+  private RawUnifiedCdcProperties rawUnifiedCdcProperties;
+
   @PostConstruct
   public void initialize() {
     logger.info("Starting unified cdc pipelines");
 
-    Optional
-            .ofNullable(cdcPipelineReaderJsonProperties)
-            .map(propertyReader::convertPropertiesToListOfMaps)
-            .orElseGet(() -> {
-              createStartSaveCdcDefaultPipelineReader(defaultCdcPipelineReaderProperties);
-              return Collections.emptyList();
-            })
-            .forEach(this::createCdcPipelineReader);
+    if (rawUnifiedCdcProperties.getReader() == null || rawUnifiedCdcProperties.getReader().isEmpty()) {
```

Comment:
Change to `rawUnifiedCdcProperties.isSOMETHING()`

Reply:
ok